util/sonar.sh

```diff
@@ -73,7 +73,7 @@ echo "Using clang-tidy in $(which clang-tidy) on $cores cores"
 type -a clang-tidy
 cat sonar/unused-headers.txt sonar/all-sources.txt \
   | $OGDF_XARGS -P "$cores" -L 10 \
-      clang-tidy -p . --quiet \
+      clang-tidy-cache $(which clang-tidy) -p . --quiet \
```

Comment:
We should update the header that clang-tidy-cache and llvm-cov are now hard dependencies when using the script on a developer machine. For llvm-cov this is actually good, as one can no longer accidentally use gcov, which leads to very hard-to-debug errors. For clang-tidy-cache, an installation link would probably be helpful if is no longer optional.